.github/ISSUE_TEMPLATE/3_ci_error_template.yml

```
@@ -0,0 +1,49 @@
+name: CI Flake report
+description: Create a report to help us improve
+title: "[Flake]: "
+labels: [triage needed, bug]
+body:
+  - type: dropdown
+    id: ci-test-type
+    attributes:
+      label: CI Test Type
+      description: In which CI test did this Flake occur?
+      options:
+        - Frontend
+        - E2E
+        - Acceptance
+        - Backend
+        - Backend associated test file check
+        - Lighthouse CI performance
+        - Lighthouse CI accessibility tests
```

```suggestion
        - Lighthouse CI accessibility
```

just for consistency with the other options

---

Done